### PR TITLE
feat: Enable automatic release

### PR DESCRIPTION
**Pull request by svetlanabrennan**
## Which problem is this PR solving?

We need to enable automatic releases. 

## Short description of the changes
- Updated renovatebot to use the renovatebot from coreint-automation
- Added `notify-failure` job to release-integration workflow
- Added renovate prs to `push_pr` workflow
- Updated security workflow actions versions
- Added `trigger_release` workflow
- Upated .gitignore file
- Updated Makefile
- Updated [release workflows](https://github.com/newrelic/k8s-metadata-injection/pull/321/files#diff-2c3b5a007a3ca5391253871b413bf6d78192fe551b28bdfe115df67aa07528b1R52) to allow v prefix on tag name. 

Note: I did not include these files in the Makefile because we were getting a build fail saying we don't have these files:
```
# Include thematic Makefiles
include $(CURDIR)/build/ci.mk
include $(CURDIR)/build/release.mk
```

## Type of change

Please delete options that are not relevant.

- [x] New feature / enhancement (non-breaking change which adds functionality)

## New Tests?

Please describe the new tests that were added (if applicable).

Not applicable. 

- [ ] This change requires changes in testing:
  - [ ] unit tests
  - [ ] e2e tests

## Checklist:

- [ ] Add changelog entry following the [contributing guide](../CONTRIBUTING.md#pull-requests)
- [ ] Tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: .github/workflows/security.yaml [hunk 1]
```diff
@@ -15,19 +16,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Run Trivy vulnerability scanner in repo mode
-        uses: aquasecurity/trivy-action@0.11.2
+        uses: aquasecurity/trivy-action@0.7.1
```
Comment (juanjjaramillo) [inline .github/workflows/security.yaml:22]: Why downgrading Trivy action? Is there a problem with the currently used version?

File: .github/workflows/security.yaml [hunk 2]
```diff
@@ -15,19 +16,20 @@ jobs:
...
         with:
           scan-type: fs
           ignore-unfixed: true
           exit-code: 1
           severity: 'HIGH,CRITICAL'
+          skip-dirs: 'build'
 
       - name: Run Trivy vulnerability scanner sarif output
-        uses: aquasecurity/trivy-action@0.11.2
+        uses: aquasecurity/trivy-action@0.7.1
```
Comment (juanjjaramillo) [inline .github/workflows/security.yaml:32]: Same comment as above

File: .github/workflows/trigger_prerelease.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  prerelease:
+    uses: newrelic/coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
+    with:
+      rt-included-files: go.mod,go.sum,build/Dockerfile
+    secrets:
+      bot_token: ${{ secrets.K8S_BOT_TOKEN }}
+      slack_channel:  ${{ secrets.K8S_SLACK_CHANNEL }}
+      slack_token: ${{ secrets.K8S_SLACK_TOKEN }}
```
Comment (juanjjaramillo) [inline .github/workflows/trigger_prerelease.yml:20]: `K8S_AGENTS_BOT_TOKEN`, `K8S_AGENTS_SLACK_CHANNEL`, `K8S_AGENTS_SLACK_TOKEN`

Comment (svetlanabrennan) [reply .github/workflows/trigger_prerelease.yml]: I've updated this. Sorry about that.

File: .github/workflows/release-integration.yml [hunk 4]
```diff
@@ -88,3 +93,15 @@ jobs:
...
+  notify-failure:
+    if: ${{ always() && failure() }}
+    needs: [docker-integration]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify failure via Slack
+        uses: archive/github-actions-slack@master
+        with:
+          slack-bot-user-oauth-access-token: ${{ secrets.K8S_SLACK_TOKEN }}
+          slack-channel: ${{ secrets.K8S_SLACK_CHANNEL }}
```
[the remark this answers is not in the document]
Comment (svetlanabrennan) [reply .github/workflows/release-integration.yml]: I've updated this. Sorry about that.

File: .github/workflows/security.yaml (hunk 2 above)
Comment (svetlanabrennan) [reply .github/workflows/security.yaml]: My mistake. I've reverted this.

**Review by juanjjaramillo — APPROVED** [review]

File: .github/workflows/release-integration.yml [hunk 5]
```diff
@@ -88,3 +93,15 @@ jobs:
...
             -t $DOCKER_IMAGE_NAME:latest \
             .
+  
+  notify-failure:
+    if: ${{ always() && failure() }}
+    needs: [docker-integration]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify failure via Slack
+        uses: archive/github-actions-slack@master
```
Comment (juanjjaramillo) [inline .github/workflows/release-integration.yml:103]: Let's pin this action to @v2 instead to avoid breaking changes and use Renovate to get updates

Comment (svetlanabrennan) [reply .github/workflows/release-integration.yml]: Done

**Pull request merged**